[warehouse] initialize_demo_db.cpp does not compile with boost-program-options 1.49.0.1

As pointed out on stackoverflow, an #include is probably missing.
http://stackoverflow.com/questions/13432201/whats-wrong-when-the-compiler-tells-me-a-variable-has-initializer-but-incomple

```
tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp: In function 'int main(int, char**)':
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:54:47: error: aggregate 'boost::program_options::options_description desc' has incomplete type and cannot be defined
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:57:14: error: 'value' is not a member of 'boost::program_options'
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:57:14: note: suggested alternative:
/usr/include/boost/bind/bind.hpp:112:25: note:   'boost::_bi::value'
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:57:55: error: expected primary-expression before '>' token
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:57:57: error: expected primary-expression before ')' token
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:58:14: error: 'value' is not a member of 'boost::program_options'
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:58:14: note: suggested alternative:
/usr/include/boost/bind/bind.hpp:112:25: note:   'boost::_bi::value'
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:58:55: error: expected primary-expression before '>' token
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:58:57: error: expected primary-expression before ')' token
/tmp/buildd/ros-groovy-moveit-ros-warehouse-0.3.25-0wheezy-20130416-2255/warehouse/src/initialize_demo_db.cpp:61:33: error: 'parse_command_line' is not a member of 'boost::program_options'
make[4]: *** [warehouse/CMakeFiles/moveit_init_demo_warehouse.dir/src/initialize_demo_db.cpp.o] Error 1
```

--- a/warehouse/warehouse/src/broadcast.cpp
+++ b/warehouse/warehouse/src/broadcast.cpp
@@ -39,6 +39,8 @@
 #include <moveit/warehouse/state_storage.h>
 
 #include <boost/program_options/cmdline.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
 #include <boost/program_options/variables_map.hpp>
 
 #include <ros/ros.h>

--- a/warehouse/warehouse/src/import_from_text.cpp
+++ b/warehouse/warehouse/src/import_from_text.cpp
@@ -42,6 +42,8 @@
 #include <moveit/robot_state/conversions.h>
 #include <eigen_conversions/eigen_msg.h>
 #include <boost/program_options/cmdline.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
 #include <boost/program_options/variables_map.hpp>
 #include <ros/ros.h>
 

--- a/warehouse/warehouse/src/initialize_demo_db.cpp
+++ b/warehouse/warehouse/src/initialize_demo_db.cpp
@@ -42,8 +42,9 @@
 #include <moveit/kinematic_constraints/utils.h>
 #include <boost/algorithm/string/join.hpp>
 #include <boost/program_options/cmdline.hpp>
-#include <boost/program_options/variables_map.hpp>
 #include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
+#include <boost/program_options/variables_map.hpp>
 #include <boost/math/constants/constants.hpp>
 #include <ros/ros.h>
 

--- a/warehouse/warehouse/src/save_as_text.cpp
+++ b/warehouse/warehouse/src/save_as_text.cpp
@@ -38,6 +38,8 @@
 #include <moveit/warehouse/state_storage.h>
 #include <moveit/warehouse/constraints_storage.h>
 #include <boost/program_options/cmdline.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
 #include <boost/program_options/variables_map.hpp>
 #include <moveit/planning_scene_monitor/planning_scene_monitor.h>
 #include <moveit/robot_state/conversions.h>

--- a/warehouse/warehouse/src/save_to_warehouse.cpp
+++ b/warehouse/warehouse/src/save_to_warehouse.cpp
@@ -40,6 +40,8 @@
 #include <moveit/planning_scene_monitor/planning_scene_monitor.h>
 #include <boost/algorithm/string/join.hpp>
 #include <boost/program_options/cmdline.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
 #include <boost/program_options/variables_map.hpp>
 #include <ros/ros.h>
 

